### DISCUSSION TITLE
"Messaging" please help me figure it out.

### DISCUSSION THREAD

**Discussion by VlaK0r** [bpatrik/pigallery2#781] [Q&A] [answered]
![изображение](https://github.com/bpatrik/pigallery2/assets/77666575/1ca0de38-b7ef-4525-b599-d538c0dbcf01)
My settings.

Hello!
Thank you for your work

Please tell me, how often are emails sent?
I haven't received any yet, maybe something is wrong in the settings?

**Comment by bpatrik** [comment, marked as answer]
Please check #683 as the site also reffers to.

The app does not send any mail by default. you need to set up jobs to do so. Only one job can send mails that is the top pick job.

**Reply by VlaK0r** [reply]
That's great!
